src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -33,49 +35,49 @@ private JobInfoHelpers() {
      *
      * @return predicate with true on apply if job contains trigger of given class
      */
-    public static Predicate<Job> withTrigger(final Class<? extends Trigger> clazz) {
-        return new Predicate<Job>() {
-            public boolean apply(Job job) {
-                return triggerFrom(job, clazz) != null;
+    public static <ITEM extends Item> Predicate<ITEM> withTrigger(final Class<? extends Trigger> clazz) {
```

Comment:
I'm not 100% sure it's a binary compatible change, same for below. Need to check it